sys-info-base.ts

```diff
@@ -212,16 +212,16 @@ export class SysInfoBase implements ISysInfo {
 		return null;
 	}
 
-	// `android -h` returns exit code 1 on successful invocation (Mac OS X for now, possibly Linux). Therefore, we cannot use $childProcess
-	private async checkAndroid(pathToAndroid: string): Promise<boolean> {
+	private async checkEmulator(): Promise<boolean> {
 		let result = false;
 		try {
-			if (pathToAndroid) {
-				let androidChildProcess = await this.$childProcess.spawnFromEvent(pathToAndroid, ["-h"], "close", {}, { throwError: false });
-				result = androidChildProcess && androidChildProcess.stdout && _.includes(androidChildProcess.stdout, "android");
-			}
+			// emulator -help exits with code 1 on Windows, so we should parse the output.
+			// First line of it should be:
+			// Android Emulator usage: emulator [options] [-qemu args]
+			const emulatorHelp = await this.$childProcess.spawnFromEvent(this.$androidEmulatorServices.pathToEmulatorExecutable, ["-help"], "close", {}, { throwError: false });
+			result = emulatorHelp.stdout.indexOf("usage: emulator") !== -1;
 		} catch (err) {
-			this.$logger.trace(`Error while checking is ${pathToAndroid} installed. Error is: ${err.messge}`);
+			this.$logger.trace(`Error while checking is emulator installed. Error is: ${err.messge}`);
```

Comment:
I think we will never trace the error message from `emulator -help`, if there is one, because we set `throwError: false` on spawnFromEvent. You can check if `result` is `false` and trace the message from `emulatorHelp.stdout` or `emulatorHelp.stderr`.